reflekt-plugin/src/main/kotlin/org/jetbrains/reflekt/plugin/scripting/ImportChecker.kt

```diff
@@ -31,17 +31,13 @@ class ImportChecker {
         )
 
         // Get all classes (each class is subtype of java.lang.Object)
+        // FIXME: logic should be revised here, as caught exception is too generic (what kind of exception was intended to catch?)
+        @Suppress("SwallowedException", "TooGenericExceptionCaught")
```

Comment:
"SwallowedException" and "TooGenericExceptionCaught" - Are last problems left and suppressed right now. But I cannot understand what kind of PitFall have you seen here.